flaskbb/templates/forum/category_layout.html

```diff
@@ -69,6 +69,20 @@
                             <a href="{{ forum.url }}">{{ forum.title }}</a>
                         </div>
 
+                        <!-- Forum Subforums -->
+                        {% if forum.subforums.all() %}
```

Comment:
Что-то я слабо себе представляю что такое all. Оно случайно не форсирует лишний запрос в БД? Я бы len сделал просто:
```suggestion
                        {% if len(forum.subforums) %}
```

Reply:
Посмотрел, у нас уже есть такие использования в коде. Плюс сходу альтернативу найти не получилось. Пофиг, пусть так пока будет